examples/user-notification.amp.html

```diff
@@ -229,6 +229,37 @@
      <button on="tap:amp-user-notification3.dismiss">I accept</button>
   </amp-user-notification>
 
+  <amp-user-notification
+          layout=nodisplay
+          id="amp-user-notification4"
+          always-show
+          data-show-if-href="http://localhost:8000/api/show?timestamp=TIMESTAMP"
```

Comment:
> Can't we have always-show be mutually exclusive to the show-ifs as enforced by the validator?
 
We can't make them exclusive since they don't mean the same things. Our "always-show" means that we should always contact the `show-if` url (if it's provided). That's not an "always show", it's more like "always contact".